src/main/java/io/confluent/connect/hdfs/HdfsSinkTask.java

```diff
@@ -116,7 +116,9 @@ public void open(Collection<TopicPartition> partitions) {
 
   @Override
   public void close(Collection<TopicPartition> partitions) {
-    hdfsWriter.close();
+    if (hdfsWriter != null) {
+      hdfsWriter.close();
```

Comment:
I think we need a warn message here to say that `close` was called on a `null` `hdfsWriter`.